compliance/google/long_stopped_instances/Projects 0.url

```diff
@@ -0,0 +1,2 @@
+[InternetShortcut]
```

Comment:
I think this file was added by mistake.

Reply:
Yes Curt I have added by mistake. I have deleted this file